client/extensions/woocommerce/state/sites/shipping-zone-locations/selectors.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { get, isObject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+const getRawShippingZoneLocations = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
Someone (I think Kelly) commented against `getApi...`. I think one word or another is needed, because `getShippingZoneLocations()` may _sound_ like something the UI would consume. Granted, it's in the `sites/shipping-zone-location/selectors.js` file instead of the UI selectors file, but I would rather be explicit about this.